test/components/kubernetes/kubelet_labels_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet baremetal
+// +build e2e
+
+package kubernetes //nolint:testpackage
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+func TestNodeHasLabels(t *testing.T) {
+	client := testutil.CreateKubeClient(t)
+
+	nodes, err := client.CoreV1().Nodes().List(metav1.ListOptions{
+		LabelSelector: "testing.io=yes,roleofnode=testing",
+	})
+	if err != nil {
+		t.Errorf("could not list nodes: %v", err)
+	}
+
+	if len(nodes.Items) == 0 {
```

Comment:
Although unlikely, this will pass if only one worker has the labels when there are several of them. Maybe we can get all nodes with label `node.kubernetes.io/node=` and make sure they're != 0 and they all contain the labels we want?

Feel free to ignore if that doesn't make sense to you.

Reply:
Hm, I can imagine we could have a case where we have more than one worker pool in the CI configuration, then this won't really work. And since we don't have access to the configuration from this test, we cannot really tell how many nodes we should have. IMO it's good enough for the time being.